### PR TITLE
PURCHASE-1470: Reuse SetupIntent for make offer

### PR DESCRIPTION
**Pull request by lilyfromseattle**
[PURCHASE-1470]

Reaction should pass confirmedSetupIntentId so that the same confirmed setup intent is used when the order is submitted a second time

[PURCHASE-1470]: https://artsyproduct.atlassian.net/browse/PURCHASE-1470

### CODE REVIEW

File: src/Apps/Order/Routes/Review/index.tsx [hunk 1]
```diff
@@ -80,12 +80,13 @@ export class ReviewRoute extends Component<ReviewProps, ReviewState> {
       },
     ],
   }))
-  async onSubmit() {
+  async onSubmit(setupIntentId?: null) {
```
Comment (ashkan18) [inline src/Apps/Order/Routes/Review/index.tsx]: is the `?` new Typescript thing? never seen it before!

Comment (ds300) [reply src/Apps/Order/Routes/Review/index.tsx]: no it's been around forever. Just means the argument is optional.

File: src/Apps/Order/Routes/Review/index.tsx [hunk 2]
```diff
@@ -170,11 +171,12 @@ export class ReviewRoute extends Component<ReviewProps, ReviewState> {
     })
   }
 
-  submitOffer() {
+  submitOffer(setupIntentId) {
```
Comment (ds300) [inline src/Apps/Order/Routes/Review/index.tsx]: this should have a type annotation. Do we always setupIntentId for an offer order now?

**Comment by artsyit** [comment]
:rocket: PR was released in v17.11.10 :rocket:

**Pull request merged**